index_po.html

```diff
@@ -171,11 +171,11 @@ <h2><b>Sobre a SCC</b></h2>
         <div class="col-md-4">
           <h4>Quem somos</h4>
           <p class="text-justify">
-            A SCC é uma empresa de software, cloud e consultoria localizada em
+            A SCC é uma empresa de consultoria em cloud localizada em
             Blumenau, Santa Catarina. Fundada por Wolfgang Unger, possuindo
             experiência em desenvolvimento de software, arquitetura e
```

Comment:
olha no novo texto aqui no git. o ingles é : SCC is a software, cloud and consulting company located in Blumenau, Santa Catarina, the 'Silicon Valley' of Brazil. Focused on Cloud, AWS & DevOps, we deliver german quality standards to our customers all over the world.
We can't get you into heaven but we can move you to the cloud!